### PR TITLE
[ACS-5304] remove unused create menu component

### PR DESCRIPTION
**Pull request by DenysVuika**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [x] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

remove unused (previously deprecated) create menu component

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
https://alfresco.atlassian.net/browse/ACS-5304

### CODE REVIEW

**Review by MichalKinas — COMMENTED** [review]
So we are just removing this feature right? Isn't this a breaking change for some clients who might be using it? Also if we're sure that we want to remove it I think you also need to remove the `Create Menu` section from `application-features.md`

**Comment by DenysVuika** [comment]
@MichalKinas it's the big "Create" button we removed with UI redesign... for those using it in the old versions they can still use it, with the new version it's not used by the app. I will update the docs, thanks

**Review by MichalKinas — APPROVED** [review]

**Review by nikita-web-ua — APPROVED** [review]

File: projects/aca-content/src/lib/components/create-menu/create-menu.component.html [hunk 1]
```diff
@@ -1,36 +0,0 @@
-<ng-container *ngIf="expanded">
-  <button
-    mat-stroked-button
-    data-automation-id="create-button"
-    [matMenuTriggerFor]="rootMenu"
-    title="{{ 'APP.NEW_MENU.TOOLTIP' | translate }}"
```
Comment (AleksanderSklorz) [inline projects/aca-content/src/lib/components/create-menu/create-menu.component.html:6]: We can probably remove this from translation files? 
image

File: projects/aca-content/src/lib/components/create-menu/create-menu.component.html [hunk 2]
```diff
@@ -1,36 +0,0 @@
...
-    [class]="{
-      'app-create-menu-secondary-button': isMainActionPresent
-    }"
-  >
-    <span class="app-create-menu__text">{{ 'APP.NEW_MENU.LABEL' | translate }}</span>
```
Comment (AleksanderSklorz) [inline projects/aca-content/src/lib/components/create-menu/create-menu.component.html:11]: the same for this translation key

**Review by AleksanderSklorz — APPROVED** [review]

File: projects/aca-content/src/lib/components/create-menu/create-menu.component.html (hunk 2 above)
Comment (DenysVuika) [reply projects/aca-content/src/lib/components/create-menu/create-menu.component.html:11]: updated, thanks

File: projects/aca-content/src/lib/components/create-menu/create-menu.component.html (hunk 1 above)
Comment (DenysVuika) [reply projects/aca-content/src/lib/components/create-menu/create-menu.component.html:6]: removed, thanks

**Pull request merged**